frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts

```diff
@@ -42,6 +42,11 @@ export const gitPage = {
         statusCode: responses.packageResponse ? 200 : 404,
         body: responses.packageResponse,
       }).as('getPackage');
+
+      cy.intercept('POST', '/api/devfile/', {
+        statusCode: responses.packageResponse ? 200 : 404,
+        body: responses.devFileResources,
+      }).as('getDevfileResources');
```

Comment:
You want return 404 if there is no devfile defined, right?

```suggestion

      cy.intercept('POST', '/api/devfile/', {
        statusCode: responses.devFileResources ? 200 : 404,
        body: responses.devFileResources,
      }).as('getDevfileResources');
```